src/resources/ble_esp32.cc

```diff
@@ -233,6 +233,21 @@ class BleReadWriteElement : public BleErrorCapableResource {
     return unvoid_cast<BleReadWriteElement*>(arg)->_on_access(ctxt);
   }
 
+  static uint16_t mbuf_total_len(os_mbuf* om) {
+    uint16_t total_len = om->om_len;
```

Comment:
I'd prefer using int for these even though I understand that it isn't going to be bigger than 64k.

```suggestion
  static int mbuf_total_len(os_mbuf* om) {
    int total_len = om->om_len;
```

Reply:
Changing it to an int, would require static casting back to uint16_t when calling os_mbuf_appendfrom.

Reply:
Yes, the result is only used for function:

```c
int os_mbuf_appendfrom(struct os_mbuf *dst, const struct os_mbuf *src, uint16_t src_off, uint16_t len)
```
`mbuf_total_len` for `len`, so using `uint16_t is better`.